tools/dataset_converters/update_infos_to_v2.py

```diff
@@ -10,6 +10,7 @@
 
 import argparse
 import copy
+import pathlib
```

Comment:
Suggest using `from pathlib import PATH` and replacing the subsequent `pathlib.Path` to `PATH`.